Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1
+
+# Stage 1 - Build the toolchain
+FROM ubuntu:18.04
```

Comment:
Did you try switching to alpine? It can be done as followup of course.

Reply:
I did try, it was not that simple though needs trial and error on my side, so I skipped.